app/helpers/patients_helper.rb

```diff
@@ -59,10 +59,14 @@ def household_size_options
   end
 
   def clinic_options
-    Clinic.where(active: true)
-          .sort_by(&:name)
-          .map { |clinic| [clinic.name, clinic.id] }
-          .unshift nil
+    clinics = Clinic.all.sort_by(&:name)
```

Comment:
New strategy here is to get the full list of clinics, then break them into active and inactive.